bundles/org.openhab.binding.goecharger/src/main/java/org/openhab/binding/goecharger/internal/GoEChargerConfiguration.java

```diff
@@ -19,10 +19,12 @@
  * The {@link GoEChargerConfiguration} class contains fields mapping thing configuration parameters.
  *
  * @author Samuel Brucksch - Initial contribution
+ * @author Reinhard Plaim - Add apiVersion
  */
 @NonNullByDefault
 public class GoEChargerConfiguration {
 
     public @Nullable String ip;
     public Integer refreshInterval = 5;
+    public Integer apiVersion = 2;
```

Comment:
If I understand this correctly, this is the default API version? For hardware revisions supporting v2, is v1 also supported? If this is the case, perhaps it would make more sense to use v1 as default for better compatibility, since I would assume older hardware revisions only supports v1?